src/renderer/components/header/AppHeader.tsx

```
@@ -69,7 +207,52 @@ export function AppHeader() {
         />
       </picture>
 
-      <nav className={[styles.nav, noDragClass].filter(Boolean).join(" ")}>
+      <nav
+        className={[styles.nav, noDragClass].filter(Boolean).join(" ")}
+        role="navigation"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)